.github/workflows/mlc_config.json

```diff
@@ -2,5 +2,5 @@
   "retryOn429": true,
   "retryCount": 5,
   "fallbackRetryDelay": "30s",
-  "aliveStatusCodes": [200, 403]
+  "aliveStatusCodes": [200, 403, 503]
```

Comment:
seems to work :)
The one link checker is green now.
And obviously the GH actions are not optimally setup when working with **branches**. I always work with forks, so it's fine for me. But now it creates 1 job for push (in the originl branch) and 1 job for PR (in the target branch).

We could remove the `on push` condition and keep `on pull_request` only

Reply:
how do you do this?